base/client.jl

```diff
@@ -419,6 +419,7 @@ function _start()
         end
 
         if repl
+            eval(Main, :(using InteractiveUtils))
```

Comment:
Wouldn't it be better to load these if the `-i` flag is set?  Then it will automatically work in IJulia and other things that use `julia -i`.